hoprd/rest-api/src/preconditions.rs

```diff
@@ -1,3 +1,4 @@
+use crate::{ApiErrorStatus, Auth, InternalState};
```

Comment:
Order of imports should actually be outside deps first, then hopr deps, then crate.